x-pack/qa/rolling-upgrade/src/test/resources/rest-api-spec/test/mixed_cluster/140_user_profile.yml

```diff
@@ -0,0 +1,33 @@
+---
+"Test User Profile feature will work in a mixed cluster":
+
+  - skip:
+      features: node_selector
+      version: " - 7.99.99"
+      reason: "https://github.com/elastic/elasticsearch/issues/86373"
```

Comment:
The BWC test fails with 7.17 node because we assert that mappings must have a key of single-mapping-name (`_doc`) in 7.17 which is not always true in a mixed cluster. Since it is an AssertionError, it does not seem to be a problem in production. I raised #86373 to discuss whether we can just remove the assertions. In the meantime, I am muting the tests for 7.x.